### DISCUSSION TITLE
1v ADC

### DISCUSSION THREAD

**Discussion by danielducu** [PricelessToolkit/ESPBell-LITE#9] [General]
Isn't 1v too low ? I mean... where do you get such low voltage in an intercom when doorbell rings ? Mine outputs 12v so it would fry this thing. Maybe get the signal from led ?

**Comment by PricelessToolkit** [comment]
1V is low, but the question is why do you think you need to connect your doorbell to ADC? It has special doorbell input marked as BELL 2-30v AC/DC.

**Comment by danielducu** [comment]
Damn, sorry man, my comment was premature lol... didn't looked over the schematic hehe.
